src/tlv/TlvCodec.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DataReaderLE } from "../util/DataReaderLE";
+import { DataWriterLE } from "../util/DataWriterLE";
+import { BitmapSchema, EnumBits } from "../util/schema/BitmapSchema";
+
+
+/**
+ * TLV element types.
+ * 
+ * @see {@link MatterCoreSpecificationV1_0} § A.7.1
+ */
+export enum TlvType {
+    SignedInt_1OctetValue = 0x00,
```

Comment:
Rename `1OctetValue` --> `U8Value` here and all similar locations?

Reply:
I tried to stay close to the name used in the specs "Signed Integer, 1-octet value" but fine to shorten it.

Reply:
Done, I just used 8 / 16 / 32 / 64 to described the variants.